sources/engine/Stride.Graphics/OpenGL/CommandList.OpenGL.cs

```diff
@@ -930,29 +929,21 @@ public unsafe MappedResource MapSubresource(GraphicsResource resource, int subRe
 
                 GL.BindBuffer(buffer.BufferTarget, buffer.BufferId);
 
-#if !STRIDE_GRAPHICS_API_OPENGLES
-                //if (mapMode != MapMode.WriteDiscard && mapMode != MapMode.WriteNoOverwrite)
```

Comment:
I don't quite know what this code does but it seems to be able to be gotten rid of since it hasn't been used since Paradox Engine